### ISSUE TITLE
Power Passthrough

### ISSUE THREAD

**Issue by Texanhick20** [mekanism/Mekanism-Feature-Requests#628]
### Describe the feature you'd like

Entangloporters can be set to be input, output, or both for power on each of it's faces.  Factories can be set to be Input on power on any of it's faces.  Either natively, or via an upgrade give Factories the ability to pass power going into them to any of their faces.   

### Describe alternatives you've considered

The only alternative is to either use 1 QE/factory or have a series of cables providing power requiring more space to build your setup.  

### Additional context

This just allows for cleaner builds where you can have factories (and other devices) side by side giving you 4 or 5 faces of the machine in question (Depending on if it's on the end or sandwiched between 2 machines) to use for various input/outputs.

**Comment by chuman72486** [comment]
I agree with this.  Being able to stack several factory machines together and only have to connect power to one and have the power pass through the machines so they all can get power from the one source would simplify the design of some factory setups.  Other mods like Applied Energistics 2 do this, where two blocks sharing a side will act as if there was a cable connecting them, passing through power (and AE2 channels) from one block to the next.
